qsprpred/extra/data/descriptors/sets.py

```diff
@@ -194,35 +194,56 @@ class PaDEL(DescriptorSet):
         descriptors (list[str]): list of PaDEL descriptor names
     """
 
-    def __init__(self, descs: list[str] | None = None, ignore_3D: bool = True):
+    _notJSON = ["_nameMapping", "_padel", "_descriptors", *DescriptorSet._notJSON]
+
+    def __init__(
+        self,
+        descs: list[str] | None = None,
+        ignore_3d: bool = True,
+        n_jobs: int | None = None,
+    ):
         """Initialize a PaDEL calculator
 
         Args:
             descs: list of PaDEL descriptor short names
-            ignore_3D (bool): skip 3D descriptor calculation
+            ignore_3d (bool): skip 3D descriptor calculation
         """
         super().__init__()
+        self.nJobs = n_jobs or os.cpu_count()
         self._descs = descs
-        self._ignore3D = ignore_3D
+        self._ignore3D = ignore_3d
+        # Initialize name mapping
+        self._initMapping()
+        # Initialize descriptors and calculator
+        if descs is None:
+            self.descriptors = None
+        else:
+            self.descriptors = descs
+
+    @property
+    def supportsParallel(self) -> bool:
+        return False
```

Comment:
Also labelled this processor as serial execution only, but the `nJobs` attribute  was added that can use the default parallelization from the PaDEL package.